flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -84,6 +84,11 @@ under the License.
 			<scope>test</scope>
 			<type>test-jar</type>
 		</dependency>
+		<dependency>
```

Comment:
This dependency is not needed, as the `${project.version}` is specified above.
(Note that `ForkableFlinkMiniCluster` was replaced by `LocalFlinkMiniCluster` in 02b852e3571e46f25fdfc79f43ceb726ddff9ba7.)